.github/workflows/history-cleanliness.yml

```diff
@@ -1,8 +1,7 @@
 name: history-cleanliness
 
 on:
-  pull_request:
-    types: [opened, reopened, synchronize, ready_for_review]
```

Comment:
this means that history-cleanliness will not run when ready for review.

also since it will not run on other pr merge, since this workflow checks if the history is semilinear, if another PR was merged in the meantime, and this is not re-triggered, this is still green.

if this is mutually exclusive to having them in order and having history-cleanliness fail the whole ci, please tell me and i will rethink the strategy